client/my-sites/stats/post-trends/style.scss

```diff
@@ -163,55 +197,117 @@
 	}
 
 	&.is-hovered {
-		border-color: var(--color-primary);
+		transform: scale(1.2);
 	}
 
 	.is-loading & {
 		animation: loading-fade 1.6s ease-in-out infinite;
 	}
 }
 
-@include breakpoint-deprecated( ">1040px" ) {
+@include breakpoint-deprecated(">1040px") {
+
 	.post-trends__scroll-left,
 	.post-trends__scroll-right {
 		display: none;
 	}
```

Comment:
We should clean up obsolete code, either in this PR or in a follow-up.

Reply:
I removed it on this PR